index.html

```diff
@@ -546,7 +549,9 @@ <h2><a href="#custom-canvas" id="custom-canvas" class="anchor">Custom Canvas</a>
     }
 
     document.getElementById('themeToggle').addEventListener('click', function (event) {
-      setTheme(activeTheme === 'light' ? 'dark' : 'light');
+      var nextTheme = activeTheme === 'light' ? 'dark' : 'light';
+      setTheme(nextTheme);
+      localStorage.setItem('theme', nextTheme);
```

Comment:
Since this is hosted on the same domain as all of my other github repo pages, this should probably be prefaced so it doesn't accidentally conflict with any of the other pages. Perhaps `canvas-confetti.theme`?